### DISCUSSION TITLE
Missing Honeypots and problem with Kibana - loading connectors

### DISCUSSION THREAD

**Discussion by jmrkvicka909** [telekom-security/tpotce#1328] [Q&A] [answered]
Hi folks, I have just downloaded ISO image, installed and update tpot. Now I have two problems/question.

1.) One of mentioned honeypots is missing https://github.com/qeeqbox/honeypots . I don't see it neither in the containers (dockers container list) nor in the images (docker image list). How can I get it?

2.) I want to get some notifications about potentional attacks to slack. I tried to use kibana connector. However when I click on Stack Management -> Connectors, I get error - Unable to load connector types

Could someone help me? Thank you.

**Comment by t3chn0m4g3** [comment, marked as answer]
#1329
